help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
 
 #### Author formats
 
-There are 13 author formats in the custom format as specified in the table below.
-
-Specifier | First Author | Second Author... | Separator | Before Last | If Shorten | Example
---- | --- | --- | --- | --- | --- | ---
-**A** | As it is in ADS | As it is in ADS | , | and | n author(s), et al. | Vinodkumar, Dharssi, I., Bally, J., Steinle, P., McJannet, D. and Walker, J.
-**G** | lastname f. i. | lastname f. i. | | and | n author(s), et al. | Vinodkumar Dharssi I. Bally J. Steinle P. McJannet D. and Walker J.
-**H** | lastname | lastname | | and | n author(s) | Vinodkumar Dharssi Bally Steinle McJannet and Walker
-**I** | lastname f. i. | f. i. lastname | , | and | first author, and xx colleagues | Vinodkumar, D. I., J., Bally, P., Steinle, D., McJannet and J. Walker
-**L** | lastname, f. i. | lastname, f. i. | , | and | first author, and xx colleagues | Vinodkumar, D. I., Bally, J., Steinle, P., McJannet, D. and Walker, J.
-**M** | lastname | lastname | , | and | n author(s), et al. | Vinodkumar, Dharssi, Bally, Steinle, McJannet and Walker
-**N** | lastname, f. i. | lastname, f. i. | , | and | first author, and xx colleagues | Vinodkumar, D. I., Bally, J., Steinle, P., McJannet, D. and Walker, J.
-**a** | As it is in ADS | As it is in ADS | , | & | n author(s), et al. | Vinodkumar, Dharssi, I., Bally, J., Steinle, P., McJannet, D. & Walker, J.
-**g** | lastname, f.i. | lastname, f.i. | , | | first author, and xx colleagues | Vinodkumar, Dharssi, I., Bally, J., Steinle, P., McJannet, D., Walker, J.
-**h** | lastname | lastname | | and | first author \emph{et al.} | Vinodkumar Dharssi Bally Steinle McJannet and Walker
-**l** | lastname, f. i. | lastname, f. i. | , | & | n author(s), et al. | Vinodkumar, Dharssi, I., Bally, J., Steinle, P., McJannet, D. \\& Walker, J.
-**m** | lastname | lastname | , | | n author(s), et al. | Vinodkumar, Dharssi, Bally, Steinle, McJannet \\& Walker
-**n** | lastname | + | | | | Vinodkumar+
-
-Note: xx is number of authors - 1.
-
-Also note: Formats *L* and *g* differ by a white space. Format *L* has a space after first initial dot, while format *g* does not. 
+There are 16 author formats in the custom format. 
+
+Author formats for **last name** only
+
+| **and**                                                                                                                        	| **format**                                                       	| **specifier** | **abbreviated**       | **example**                                                                                                                    	| **example abbreviated %3.2**                                                            	|
```

Comment:
* In both tables, capitalize the first letter of each of the column headers, to match the other tables and separate the columns more visually. 
* The "separator" column was removed here. I don't think it should be added back, as the columns are already pretty narrow, but can you add a note to the "format" column? The comma separator is standard, so you can just add a note if no separator is used.
* Depending on how much trouble it is, it'd be good to have the "specifier" column moved to be the first column (to match the other tables on the page) and for the "abbreviated" column to be moved in between the "example" and "example abbreviated %3.2" columns, to be closer to where it's being used.

Reply:
regarding separator, turned out that was my invention, lol, in classic there is only `,`. I had gotten creative and revised some of the formats to have `;`. Alberto made me changed them all back. Back to classic format. However, we now have separator replacement, that user can specify what they want to replace the comma with.

Don't you think specifier be in the middle is better, so that user can look to the left see the format, look to the right see the examples. kind of get their focus in the middle. That is how I am explaining it.

Reply:
Ah, I understand now, about the separator. But it looks like %H and %h aren't using commas as separators (at least in the examples). There are just spaces between the authors. Is that correct? If so, I think you can just add a note to the format field for these two formats to clarify, similar to the notes you have for %g and %e

And I'll defer to your reasoning for leaving the column placement as is ;)